src/foam/u2/wizard/WizardletSearchController.js

```diff
@@ -96,9 +96,17 @@ foam.CLASS({
           if ( seen.includes(cls) ) return;
           seen.push(cls);
           for ( let p of cls.getAxiomsByClass(foam.core.Property) ) {
-            var visibilityEnum = p.visibility instanceof Function
-              ? p.visibility.apply(w.data)
-              : p.visibility;
+            var visibilityEnum = '';
+            if ( p.visibility instanceof Function ) {
+              var args = p.visibility.toString().match(/\((?:.+(?=\s*\))|)/)[0]
```

Comment:
Comment will prolly help to say that we're grabbing the arguments from the visibility method